eng/common/pipelines/templates/steps/validate-all-packages.yml

```diff
@@ -4,7 +4,7 @@ parameters:
   ConfigFileDir: $(Build.ArtifactStagingDirectory)/PackageInfo
 
 steps:
- - ${{ if and(ne(variables['Skip.PackageValidation'], 'true'), eq(variables['System.TeamProject'], 'internal')) }}:
+ - ${{ if and(ne(variables['Skip.PackageValidation'], 'true'), and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'))) }}:
```

Comment:
@praveenkuttappan do you recall why you added the `not(endsWith(variables['Build.Repository.Name'], '-pr'))` condition below? Does this not work at all in the private repos?

Reply:
@benbp is conditions like these that make me worry about the test pipeline being in internal.